discussions.go

```diff
@@ -727,6 +727,7 @@ type CreateMergeRequestDiscussionOptions struct {
 	Body      *string       `url:"body,omitempty" json:"body,omitempty"`
 	CreatedAt *time.Time    `url:"created_at,omitempty" json:"created_at,omitempty"`
 	Position  *NotePosition `url:"position,omitempty" json:"position,omitempty"`
+	CommitID  *string       `url:"commit_id,omitempty" json:"commit_id,omitempty"`
```

Comment:
If you can place this one above `Possition` so it's alphabetically ordered following the ordering in the docs, I'll take it...

Reply:
Thanks @svanharmelen .  Your comment was a bit confusing.  

In the version I just pushed, I placed `CommitID` above `CreatedAt` to put it in alphabetical order, not `Position`.  I hope that's okay.  If not let me know and I can simply push a version that does put `CommitID` above `Position`.